app/controllers/reviewables_controller.rb

```diff
@@ -196,6 +196,26 @@ def destroy
     render json: success_json
   end
 
+  def scrub
```

Comment:
```suggestion
  def scrub_reviewable_user
```

A slightly more verbose suggestion since the  controller action only targets reviewables of `ReviewableUser` type.

Reply:
I went back and forth a bit on this - I'm inclined to use the more generic name, so it's easier to expand to other reviewable types if we need to in the future.

Perhaps it would be better to check if the reviewable model has a `scrub` method to call, rather than just hardcoding the `ReviewableUser` type.

Reply:
> Perhaps it would be better to check if the reviewable model has a scrub method to call, rather than just hardcoding the ReviewableUser type.

I think this will be better. We could also just have a constant to define scrubbable reviewables.